src/Types/POSIXTimeRange.purs

```diff
@@ -48,19 +100,177 @@ instance Show a => Show (UpperBound a) where
 -- | that the endpoints may or may not be included in the interval.
 -- |
 -- | The interval can also be unbounded on either side.
-newtype Interval a = Interval { from :: LowerBound a, to :: UpperBound a }
+newtype Interval a = Interval { ivFrom :: LowerBound a, ivTo :: UpperBound a }
```

Comment:
Why the change here?

Reply:
I do prefer `from` and `to` but it's to help people when they copy & paste - less changes. Happy to change back if you want.